rtgui/spot.cc

```diff
@@ -185,6 +186,25 @@ void Spot::resetPressed()
     }
 }
 
+/**
+ * Release anything that's currently being dragged.
+ */
+void Spot::releaseEdit()
+{
+    Geometry *loGeom = getVisibleGeometryFromMO (lastObject);
+
+    if (!loGeom) {
+        EditSubscriber::action = EditSubscriber::Action::NONE;
+        return;
+    }
+
+    loGeom->state = Geometry::NORMAL;
+    sourceIcon.state = Geometry::NORMAL;
+    EditSubscriber::action = EditSubscriber::Action::NONE;
```

Comment:
Can be moved up. It's the same in the `!loGeom` case.

Reply:
Done.